packages/x-charts/src/internals/plugins/corePlugins/useChartId/useChartId.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { ChartPlugin } from '../../models';
+import { UseChartIdSignature } from './useChartId.types';
+import { createChartDefaultId } from './useChartId.utils';
+
+export const useChartId: ChartPlugin<UseChartIdSignature> = ({ params, store }) => {
```

Comment:
This is taken from the tree-view.

It allows the user to pass a custom id that override the default one. We don't use that feature we don't have an id prop

Reply:
probably would be good to add it in the future though